src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/dataset/S3DatasetPackage.kt

```diff
@@ -1,16 +1,21 @@
-package com.atlassian.performance.tools.awsinfrastructure
+package com.atlassian.performance.tools.awsinfrastructure.api.dataset
```

Comment:
We should mention new API in the changelog

Reply:
We should add a bot to remind about that 😄 .

Reply:
I've added an entry to the CHANGELOG.md.

Reply:
I've added an entry to the CHANGELOG.md.